templates/telco/osm/index.html

```diff
@@ -186,6 +186,36 @@ <h2>Enterprise support</h2>
       </p>
     </div>
   </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator" />
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <h2>OSM Community</h2>
+      <p>ETSI OSM community is growing rapidly with 147 members today. Canonical is driving the ETSI OSM project by leading major positions:
+      </p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Charmed OSM design and delivery</li>
+        <li class="p-list__item is-ticked">MANO stack enterprise support</li>
+        <li class="p-list__item is-ticked">Fully managed OSM</li>
+        <li class="p-list__item is-ticked">Onboarding network functions</li>
```

Comment:
This is different in the copy docs

Reply:
Sorry! Missed updating that part. It's been updated :)